### ISSUE TITLE
Invalid characters in output owl

### ISSUE THREAD

**Issue by ozgunbabur** [PathwayCommons/cpath2#298]
I am getting an owl file with the following graph query.

http://beta.pathwaycommons.org/pc2/graph?source=CDK4&source=CDKN2A&kind=commonstream&direction=DOWNSTREAM

But Paxtools cannot load it because an author name in the file has invalid (or non-ascii, or whatever) characters in it. I am not sure if that is a data import problem or a BioPAX generation problem.

**Comment by IgorRodchenkov** [comment]
Have you got that biopax with curl or wget or programmatically?

Otherwise, sounds like a re-occurrence of the old paxtools bug...

IR

**Comment by ozgunbabur** [comment]
I am just clicking on the link and Chrome downloads the owl. I try to open the owl with ChiBE, but the Paxtools inside throws the error. I also tried outside of ChiBE, using Main in paxtools-console. I got:

Exception in thread "main" org.biopax.paxtools.util.BioPaxIOException: XMLStreamException ParseError at [row,col]:[8747,74]
Message: Invalid byte 2 of 3-byte UTF-8 sequence.; Line number = 8747
Column number = 74
System Id = null
Public Id = null
Location Uri= null
CharacterOffset = 743620

	at org.biopax.paxtools.io.SimpleIOHandler.createAndBind(SimpleIOHandler.java:314)
